### ISSUE TITLE
Padding above and below pages is too large

### ISSUE THREAD

**Issue by GeekPro101** [dowjcr/web-frontend#3]
Padding at the moment is quite large and feels like there's too much empty space:

![image](https://github.com/dowjcr/web-frontend/assets/14115426/99df2c69-75d8-4a72-9668-2783b9cfc53b)

Reducing this to py-10 (2.5rem):

**Comment by GeekPro101** [comment]
There's also probably a bit too much padding at the bottom, but this is not as noticeable

**Comment by apple-phi** [comment]
Agreed. Had a play around with 2.5rem vs 3rem; 2.5rem feels slightly squashed but I'm happy to have it like this for now.
